pkgs/misc/drivers/xp-pen-deco-01-v2-driver/default.nix

```
@@ -0,0 +1,57 @@
+{ lib, stdenv, fetchzip, libusb, glibc, libGL, xorg, qtx11extras, wrapQtAppsHook, autoPatchelfHook, ... }:
+
+stdenv.mkDerivation {
+  pname = "xp-pen-deco-01-v2-driver";
+  version = "3.2.3.220323-1";
+  src = fetchzip {
+    url = "https://www.xp-pen.com/download/file/id/1936/pid/440/ext/gz.html#.tar.gz";
+    sha256 = "sha256-n/yutkRsjcIRRhB4q1yqEmaa03/1SO8RigJi/ZkfLbk=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    libusb
+    xorg.libX11
+    xorg.libXtst
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXinerama
+    glibc
+    libGL
+    stdenv.cc.cc.lib
+    qtx11extras
+  ];
+
+  installPhase = let dataDir = "var/lib/xppend1v2"; in ''
+    runHook preInstall
+
+    mkdir -p $out/{opt,bin}
+    cp -r App/usr/lib/pentablet/{pentablet,resource.rcc,conf} $out/opt
+    chmod +x $out/opt/pentablet
+    cp -r App/lib $out/lib
+    sed -i 's#usr/lib/pentablet#${dataDir}#g' $out/opt/pentablet
+    makeWrapper $out/opt/pentablet $out/bin/xp-pen-deco-01-v2-driver \
```

I don't think the version should be apart of the executables name, that might break configurations in the future. In my opinion we should try to match upstreams executable names to avoid confusion.

Also, you could include the wrapper arguments from `wrapQtAppsHook` in this to avoid double wrapping the executable:
```suggestion
    makeWrapper $out/opt/pentablet $out/bin/pentablet \
      "${qtWrapperArgs[@]}"
```
You should also set `dontWrapQtApps = true;` with that.

I do think it would be best to move the `makeWrapper` functionality to `postFixup` though, as to ensure all setup hooks are properly propagated.